### PR TITLE
Added lastExecutionRecordCount in Collector, and persist it into DB.

### PR DESCRIPTION
**Pull request by danielyhuang**

### CODE REVIEW

**Comment by rvema** [comment]
Please update your branch

**Comment by rvema** [comment]
This is a breaking change, we would need to think of a different way.

File: src/main/java/com/capitalone/dashboard/collector/CollectorTask.java [hunk 1]
```diff
@@ -62,13 +62,14 @@ public final void run() {
             LOGGER.info("Starting Collector={}", collectorName);
             long start = System.currentTimeMillis();
             collect(collector);
+            long count = collector.getLastExecutionRecordCount();
```
Comment (nireeshT) [inline src/main/java/com/capitalone/dashboard/collector/CollectorTask.java:65]: You don’t have to get and set count back to collector.  It is already available

Comment (danielyhuang) [reply src/main/java/com/capitalone/dashboard/collector/CollectorTask.java:65]: removed unnecessary setter call

**Pull request merged**